bin/serverless.js

```diff
@@ -8,25 +8,29 @@ const logError = require('../lib/classes/Error').logError;
 const uuid = require('uuid');
 const initializeErrorReporter = require('../lib/utils/sentry').initializeErrorReporter;
 
-Error.stackTraceLimit = Infinity;
+(() => {
+  if (Number(process.version.split('.')[0].slice(1)) >= 8) {
+    const componentsCli = require('@serverless/cli');
```

Comment:
I wouldn't call it `componentsCli` but `cli` or `serverlessCli`.

Reply:
done